app/services/github_service.rb

```diff
@@ -0,0 +1,26 @@
+class GithubService
```

Comment:
Generally looks like you're doing what you need to do here. 👍 

Might not be the time now, but wondering if we could have put the Service and OAuth pieces into a single class. On a related note, if you initialize before calling followers, etc. you shouldn't need to define your `client_id` and `client_secret` in each of these methods.